soc/arm/actions/atb110x/soc_reset.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019 Actions (Zhuhai) Technology Co., Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Peripheral reset configuration macros for ATB110X
+ */
+
+#ifndef _ACTIONS_SOC_RESET_H_
+#define _ACTIONS_SOC_RESET_H_
+
+#define RESET_ID_DMA		0
```

Comment:
Should probably encode these in DTS.

Reply:
I'm not sure how to pass it since there is no proper reset driver. I'd keep this until we fix later.